commands/shortenlink.js

```diff
@@ -0,0 +1,28 @@
+const { SlashCommandBuilder } = require("discord.js")
+const axios = require("axios").default
+
+module.exports = {
+  data: new SlashCommandBuilder()
+    .setName("shortenlink")
+    .setDescription("Shorten the url passed by the user.")
+    .addStringOption((option) =>
+      option.setName("url").setDescription("URL to input.").setRequired(true)
+    ),
+  async execute(interaction) {
+    await axios({
+      method: "post",
+      headers: {
+        "Content-Type": "application/json",
+      },
+      url: "https://gotiny.cc/api",
+      data: JSON.stringify({ input: interaction.options.getString("url") }),
+      responseType: "json",
+    })
+      .then((response) => {
+        interaction.reply("Shortlink: gotiny.cc/" + response.data.code)
```

Comment:
Please add https:// in front of the link and change the variable to `response.data[0].code` because the response.data object is an array
```js
interaction.reply("Shortlink: https://gotiny.cc/" + response.data[0].code)
```